Source/JavaScriptCore/inspector/protocol/CSS.json

```diff
@@ -462,6 +462,14 @@
                 { "name": "fontFamilyNames", "type": "array", "items": { "type": "string" }, "description": "Supported system font families." }
             ]
         },
+        {
+            "name": "getLoadedFonts",
+            "description": "Returns all loaded fonts (system and custom).",
```

Comment:
we already have `CSS.getSupportedSystemFontFamilyNames`.  if this also includes system font family names, perhaps we should remove that in favor of this?  or maybe just make this load custom fonts (i.e. `CSS.getLoadedCustomFonts`)?

Reply:
Wouldn't we still need `CSS.getSupportedSystemFontFamilyNames` for loading the font family names of system fonts? It is different from `CSS.getLoadedCustomFonts` where it is any custom font. I will possibly include both in retrieving the font data.

Reply:
Are you suggesting to combine them? I don't see `CSS.getSupportedSystemFontFamilyNames` in use in the frontend.

Reply:
If not, I am going to remove it. I don't see it.